src/town_cmd.cpp

```diff
@@ -2879,15 +2883,26 @@ static bool TryBuildTownHouse(Town *t, TileIndex tile)
 		/* Special houses that there can be only one of. */
 		t->flags |= oneof;
 
-		BuildTownHouse(t, tile, hs, house, random_bits);
+		/* Should the house be protected? */
```

Comment:
If you would name your variable `house_protected`, this comment would be pointless. It now isn't as `protect` is very non-descriptive :D See how proper naming can help you avoid writing silly comments? :D :)